assets/js/util/helpers.js

```diff
@@ -35,6 +35,8 @@ import { createElement, Fragment } from '@wordpress/element';
  *
  * @param {Component} NewComponent The new component to append to the filtered component.
  *
+ * @return {ReactElement} Filtered component with appended component.
```

Comment:
I believe we should be using `WPElement` instead since React is essentially provided through we're using `@wordpress/element`, but I'm not 100% sure about that now that we're using our own bundles. 

@tofumatt ?

Reply:
See above; I think `Element` is the way to go.